public_html/wp-content/mu-plugins/blocks/source/blocks/organizers/block-controls.js

```diff
@@ -61,11 +61,11 @@ export class BlockControls extends Component {
 			default :
 				output = (
 					<Placeholder
-						className={ classnames( 'wordcamp__edit-placeholder', 'has-no-mode' ) }
+						className={ classnames( 'wordcamp-block__edit-placeholder', 'has-no-mode' ) }
```

Comment:
Corey and I have been using a `MUST`, `SHOULD`, `MAYBE` convention during reviews [to try and make expectations clear](https://www.brandonsavage.net/the-pitfalls-of-code-review-and-how-to-fix-them/).

"I think..."  is still implied, though, so even if I said `MUST`, that would just mean, I "I think this is a blocker", and you're always welcome to push back if you disagree.

----

`SHOULD` - I think this needs a wider discussion, and feels unrelated to this PR. I'd separate it out into a new PR so we can discuss it there.

See https://github.com/WordPress/wordcamp.org/issues/96 / #123

Reply:
Makes sense. The standardization was more important before I slightly-refactored the PlaceholderSpecificMode component, since it was clearly supposed to be using the same prefix between the two. I'll push that to another PR so we can pick which prefix to use.